Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -29,24 +30,64 @@ extension BrowserViewController {
+                enableInstalledVPN: { [unowned menuController] in
+                    /// Donate Enable VPN Activity for suggestions
+                    let enableVPNActivity = ActivityShortcutManager.shared.createShortcutActivity(type: .enableBraveVPN)
+                    menuController.userActivity = enableVPNActivity
+                    enableVPNActivity.becomeCurrent()
+                }
+            )
+            
+            // Add Brave Talk and News options only in normal browsing
+            if !PrivateBrowsingManager.shared.isPrivateBrowsing {
+                MenuItemButton(
+                    icon: #imageLiteral(resourceName: "menu-brave-talk").template,
+                    title: "Brave Talk",
+                    subTitle: Strings.PrivacyFeature.braveTalkItemDescription) { [weak self] in
+                    guard let self = self, let url = URL(string: "https://talk.brave.com/") else { return }
+                    
+                    self.popToBVC()
+                    self.finishEditingAndSubmit(url, visitType: .typed)
+                }
+                
+                // Show Brave News if it is first launch and after first launch If the new is enabled
+                if Preferences.General.isFirstLaunch.value || (!Preferences.General.isFirstLaunch.value && Preferences.BraveNews.isEnabled.value) {
+                    MenuItemButton(
+                        icon: #imageLiteral(resourceName: "menu_brave_news").template,
+                        title: "Brave News",
+                        subTitle: Strings.PrivacyFeature.braveNewsItemDescription) { [weak self] in
+                        guard let self = self, let newTabPageController = self.tabManager.selectedTab?.newTabPageViewController  else {
+                            return
+                        }
 
-            MenuItemButton(icon: #imageLiteral(resourceName: "menu-history").template, title: Strings.historyMenuItem) { [unowned self, unowned menuController] in
-                let vc = HistoryViewController(isPrivateBrowsing: PrivateBrowsingManager.shared.isPrivateBrowsing, historyAPI: historyAPI)
-                vc.toolbarUrlActionsDelegate = self
-                menuController.pushInnerMenu(vc)
-            }
-            MenuItemButton(icon: #imageLiteral(resourceName: "menu-downloads").template, title: Strings.downloadsMenuItem) { [unowned self, unowned menuController] in
-                let vc = DownloadsPanel(profile: self.profile)
-                menuController.pushInnerMenu(vc)
+                        self.popToBVC()
+                        newTabPageController.scrollToBraveNews()
+                    }
+                }
             }
-            MenuItemButton(icon: #imageLiteral(resourceName: "playlist_menu").template, title: Strings.playlistMenuItem) { [weak self] in
+            
+            MenuItemButton(
+                icon: #imageLiteral(resourceName: "playlist_menu").template,
+                title: "Brave Playlist",
```

Comment:
This is translated i believe, for some of those you can reuse existing copy

Reply:
See https://github.com/brave/brave-ios/pull/4424#discussion_r741417115

Reply:
Fix at https://github.com/brave/brave-ios/pull/4424/commits/156d50f94bf8089de48e348deca6ec26c6897cdb